### PR TITLE
Refs [TASK][34091] Create Details Game Screen

### PR DESCRIPTION
**Pull request by Hoang-Kelvin**
## Related Tickets
- [#Task 34091: Create Details Game Screen](https://edu-redmine.sun-asterisk.vn/issues/34091)
## WHAT
-Create Detail Game UI
## Evidence (Screenshot or Video)
image
## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/project/gamedb/ui/details/DetailsFragment.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+    private fun initPresenter() {
+        val repository = Repositories.getRemoteRepository(GameRemoteDataSource.getInstance())
+        detailsGamePresenter = DetailsPresenter(this, repository)
+    }
+
+    override fun showGameDetail(game: GameDetail) {}
+
+    override fun showError(error: String) {
+        context?.showToast(error, 1)
```
Comment (thonc-2135) [inline app/src/main/java/com/project/gamedb/ui/details/DetailsFragment.kt]: hard code

File: app/src/main/java/com/project/gamedb/ultis/ViewExtentions.kt [hunk 2]
```diff
@@ -11,3 +15,21 @@ fun View.OnClickListener.assignViews(vararg views: View?) {
...
+
+                override fun onResourceReady(
+                    resource: Drawable,
+                    transition: com.bumptech.glide.request.transition.Transition<in Drawable>?
+                ) {
+                    this@loadDrawable.background = resource
+                }
+            })
+    }
+
```
Comment (thonc-2135) [inline app/src/main/java/com/project/gamedb/ultis/ViewExtentions.kt]: remove blank

File: app/src/main/res/layout/fragment_details.xml [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+            android:text="@string/string_games"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/sp_20"
+            app:layout_constraintBottom_toTopOf="@id/metaScore"
+            app:layout_constraintEnd_toEndOf="@id/metaScore"
+            app:layout_constraintStart_toStartOf="@id/metaScore"
+            app:layout_constraintTop_toTopOf="@id/imageInfoBar" />
+
+        <TextView
+            android:id="@+id/Rating"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_details.xml]:
```suggestion
            android:id="@+id/textRating"
```

File: app/src/main/res/layout/fragment_details.xml [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/dp_35"
+            android:layout_marginBottom="@dimen/dp_20"
+            android:text="@string/string_meta_score"
+            android:textColor="@android:color/white"
+            app:layout_constraintBottom_toBottomOf="@id/imageInfoBar"
+            app:layout_constraintEnd_toEndOf="@id/imageInfoBar" />
+
+        <TextView
+            android:id="@+id/about"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_details.xml]: mấy cái này nữa nhé , e k cmt hết

File: app/src/main/res/layout/fragment_details.xml [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+            android:text="@string/string_updated"
+            android:textColor="@android:color/white"
+            app:layout_constraintEnd_toEndOf="@id/textUpdatedDetail"
+            app:layout_constraintStart_toStartOf="@id/textUpdatedDetail"
+            app:layout_constraintTop_toBottomOf="@id/textPlaytime" />
+
+        <TextView
+            android:id="@+id/textUpdatedDetail"
+            android:layout_width="0dp"
+            android:layout_height="11dp"
```
Comment (thonc-2135) [inline app/src/main/res/layout/fragment_details.xml]: hard code

File: app/src/main/java/com/project/gamedb/ui/details/DetailsFragment.kt [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+    override fun startComponents() {
+        initPresenter()
+    }
+
+    private fun initPresenter() {
+        val repository = Repositories.getRemoteRepository(GameRemoteDataSource.getInstance())
+        detailsGamePresenter = DetailsPresenter(this, repository)
+    }
+
```
Comment (thonc-2135) [inline app/src/main/java/com/project/gamedb/ui/details/DetailsFragment.kt]: sắp xếp lại thứ tự a ơi, mấy cái overide để trên, hàm viết thêm vứt xuống dưới

**Review by thonc-2135 — APPROVED** [review]

File: app/src/main/res/values/strings.xml [hunk 7]
```diff
@@ -23,4 +30,6 @@
     <string name="text_xbox">xbox</string>
     <string name="text_nintendo">nintendo</string>
     <string name="text_mobile">mobile</string>
+
+
```
Comment (huytq-2422) [inline app/src/main/res/values/strings.xml:34]: remove

**Pull request merged**